CONTRIBUTING.rst

```diff
@@ -90,13 +90,20 @@ including testing other Python versions with tox::
 
   $ black .
 
-7. Commit your changes and push your branch to GitHub::
+7. [Optional] If you changed something related to docs, make sure it compiles properly
```

Comment:
Compile the docs is not optional. You need to make sure that the docs were generated correctly before sending the PR. Also includes the need to install requirements-docs.txt so it is possible to compile it.

> If you changed something related to docs, **make sure it compiles properly**:

Reply:
@ejulio Do you want to include this compile as a CI check here so if it's failed, it's visible?

Reply:
Otherwise, maybe we could add the docs to CI in a different pull request. Scrapy could serve as a starting point here, documentation warnings are treated as errors by the CI there.

Reply:
It is a good idea to include the build in CI. I created #169 to track this separately from this PR.
Anyway, add to the instructions that build the docs is not an optional step is needed.

Reply:
hey @rennerocha , updated :)